module/main.go

```diff
@@ -12,11 +14,42 @@ import (
 	viamcartographer "github.com/viamrobotics/viam-cartographer"
 )
 
+// Versioning variables which are replaced by LD flags.
+var (
+	Version     = "development"
+	GitRevision = ""
+)
+
 func main() {
 	utils.ContextualMain(mainWithArgs, golog.NewLogger("cartographerModule"))
 }
 
 func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error {
+	var versionFields []interface{}
+	// print version on statup
+	// exit 0 if --version or -version argv is provided
+	// NOTE: utils.ParseFlags can't be used w/o a nolint as it
```

Comment:
update comment to not discuss nolint that no longer exists

Reply:
done